app/src/ui/history/compare.tsx

```diff
@@ -18,6 +18,7 @@ import { FancyTextBox } from '../lib/fancy-text-box'
 import { OcticonSymbol } from '../octicons'
 
 enum SelectedTab {
+  None = -1,
```

Comment:
I feel that *None* does a better job of communicating intent than *Default* or defining the type of `selectedTab` as `readonly selectedTab: SelectedTab | null`.